api/src/org/apache/cloudstack/api/response/SslCertResponse.java

```diff
@@ -38,6 +40,7 @@
 
     @SerializedName(ApiConstants.CERTIFICATE)
     @Param(description = "certificate")
+    @LogLevel(Log4jLevel.Off)
```

Comment:
I suppressed cert. related data from getting logged. But you are right they are not sensitive.